tests/test_bioimg.py

```diff
@@ -19,9 +19,11 @@ def test_validate_io_paths_accepted(self):
                 ["s3://test_out/b", "s3://test_out/d"],
             ),
         }
-        for test_name, (source, dest) in accepted_pairs.items():
-            with self.subTest(f"case: {test_name}"):
-                validate_io_paths(source, dest)
+        register = {"true": True, "false": False}
+        for _, r in register.items():
+            for test_name, (source, dest) in accepted_pairs.items():
```

Comment:
@ktsitsi what would you think about moving these loops to **outside** the test function using `pytest.parameterize()`? It's more powerful and more clear than `subTest`.

Reply:
Indeed @sgillies and we are using `pytest.parametrize` in other repos. The tests for bioimg in this repo need to be redesigned in any case and there is a ticket for that. I followed this path so the tests match the test style of other files and subtest was a suggestion in a previous PR comment. But I am positive in changing them in a separate PR.

Reply:
Agreed on this. The tests in this package are written in classic `unittest` style mostly as an artifact of history and we could generally get a lot of mileage out of moving things to use more pytest infrastructure.